doofinder-for-woocommerce/includes/class-thumbnail.php

```diff
@@ -18,29 +18,14 @@ class Thumbnail {
 	 */
 	private $post;
 
-	/**
-	 * Prepare the thumbnail size to be used when submitting
-	 * the images to the Doofinder API.
-	 *
-	 * Default WP thumbnail size (150x150) if it's available,
-	 * but if it's not we'll register our own.
-	 */
-	public static function prepare_thumbnail_size() {

```

Comment:
Here you remove the prepare_thumbnail_size() function but in line [102](https://github.com/doofinder/doofinder-woocommerce/pull/174/files#diff-6de0a4988b16dc6efb250da24e08311b61d1c7c89397608a8100a58f9f5c0391R102 ) you are using it. That generates inconsistency.

Reply:
Thanks, I've removed this function call now.